db/db_impl.h

```diff
@@ -965,10 +968,18 @@ class DBImpl : public DB {
 
   Status SwitchMemtable(ColumnFamilyData* cfd, WriteContext* context);
 
+  void SelectColumnFamiliesForAtomicFlush(autovector<ColumnFamilyData*>* cfds,
+                                          bool check_immutable_memtables);
```

Comment:
oh btw, I didn't fully understand this part. Why don't we need to consider immutable memtables when the flush reason is full write buffer?

Reply:
Good point. Let me do some experiments and get back to you.

Reply:
I did some investigation. When atomic flush is enabled, we need to check immutable memtables as well so that we do not miss flushing a column family.